diorama/src/ev-charging/ev-charging.py

```diff
@@ -40,11 +40,17 @@ def ack_topic(cmd_topic):
 # topic base config
 protocol = str(config['connection']['protocol'])
 service_api_key = str(config['ev_parking']['ev_service_api_key'])
```

Comment:
@Solargi do you still need `service_api_key` the value is the same as `ev_api_key`

Reply:
service_api_key removed